### PR TITLE
New package: Stacher.youtube-dl version 6.0.14

### PR DESCRIPTION
**Pull request by cuccurese2010**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111393&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
This package's desktop shortcut references a removed file: 
image

**Comment by cuccurese2010** [comment]
> This package's desktop shortcut references a removed file: image
I used winget install -m . locally  and i didn't have any problem. 
I just receive this message locally from windows:

Capture

**Comment by stephengillie** [comment]
> I used winget install -m . locally  and i didn't have any problem.

Does this package have any dependencies?

> I just receive this message locally from windows:
> 
> Capture

Yes, this happens sometimes, including when an application has too few "good" ratings. To accept all risks and proceed with install, click "More info" then "Run anyway".

**Comment by cuccurese2010** [comment]
> Does this package have any dependencies?
No dependencies i'm using windows sandbox.

File: manifests/s/Stacher/youtube-dl/6.0.14/Stacher.youtube-dl.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Stacher.youtube-dl
+PackageVersion: 6.0.14
+PackageLocale: en-US
+Publisher: StacherIO
+PackageName: UI for youtube-dl
+License: StacherIO
```
Comment (Trenly) [inline manifests/s/Stacher/youtube-dl/6.0.14/Stacher.youtube-dl.locale.en-US.yaml:9]:
```suggestion
License: CC BY-NC-ND 4.0
```
image

File: manifests/s/Stacher/youtube-dl/6.0.14/Stacher.youtube-dl.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Stacher.youtube-dl
+PackageVersion: 6.0.14
+PackageLocale: en-US
+Publisher: StacherIO
+PackageName: UI for youtube-dl
```
Comment (Trenly) [inline manifests/s/Stacher/youtube-dl/6.0.14/Stacher.youtube-dl.locale.en-US.yaml:8]:
```suggestion
PackageName: Stacher
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by BrandonWanHuanSheng** [comment]
#11634

**Comment by BrandonWanHuanSheng** [comment]
#111634

**Comment by cuccurese2010** [comment]
> #111634

closed and reopened.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**